infoextract.js

```diff
@@ -48,7 +47,15 @@ module.exports = {
 	},
 
 	getService(link) {
+		if (typeof link !== "string") {
+			return false;
+		}
+
 		let srcUrl = url.parse(link);
+		if (srcUrl.host === null) {
+			return false;
+		}
```

Comment:
Is this temporary until we add more services? If not, I would make sure that either the function returns empty strings instead of false, or raises an error.

Reply:
It's to ensure that `link` is a valid URL